site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -4,6 +4,8 @@ api: offscreen
 
 ## Manifest
 
+Offscreen documents are invisible webpages with their own DOM that can be created and managed using the `offscreen` API. Requesting this permission allows the extension to use DOM APIs without obtrusively opening new windows or tabs that interrupt the user experience. 
```

Comment:
Thoughts on moving this above the manifest header?

Reply:
I agree. This is a basic offscreen definition. the place for it is here: https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/api/offscreen.idl;l=5

Reply:
@IanStanion-google, as requested, here's the Gerrit PR: https://chromium-review.git.corp.google.com/c/chromium/src/+/4440738

It doesn't include your second sentence. I think it's more appropriate here.